requests_toolbelt/adapters/appengine.py

```diff
@@ -59,6 +79,24 @@ def init_poolmanager(self, connections, maxsize, block=False):
         self.poolmanager = _AppEnginePoolManager(self._validate_certificate)
 
 
+class InsecureAppEngineAdapter(AppEngineAdapter):
+    """A variant of the the transport adapter for Requests to use urllib3's GAE
+    support that does not validate certificates by default. Use with caution!
+
+    See :class:`AppEngineAdapter` for further details.
```

Comment:
We should probably add to the docstring:

```rst

.. note::

    The ``validate_certificate`` keyword argument will not be honored here and is not part of the signature because we always force it to ``False``.
```